runtime/strings/bytes.cpp

```diff
@@ -10,6 +10,8 @@
 
 extern "C" {
 
+#undef get_ui
+#define get_ui(x) get_ui_named(x, __PRETTY_FUNCTION__)
```

Comment:
My bad here when specifying the issue; we probably want to use `__func__` rather than `__PRETTY_FUNCTION__` as the latter is an extension rather than part of the standard.